lessons/Algebra2/exploring-linear-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,265 @@
+
+=== Launch
+
+[.lesson-instruction]
+In Pyret, make a scatter plot showing the the relationship between `college-or-higher` and `median-household-income`.
+
+@center{@image{images/college-v-income.png}}
+
+This scatter plot appears to show a positive, linear relationship: states with higher percentages of college graduates tend to have higher median household incomes.
+
+[.lesson-instruction]
+Suppose the United States were to add a new state. **Based on the data for the existing 50 states, what median household income would you predict, if exactly 50% of the new state's citizens had attended college?** What about 90%? 10%
+
+Let students discuss, and explain their thinking. If possible, mark off a single point for each of the hypothetical percentages, then connect those points to show a straight line. Note that some of these new points would require changing the x- and y-axes of our display!
+
+When we see patterns in data, we can use those patterns to __make predictions__ based on that data. We can even draw a line to show all the possible predictions at once! These predictions represent our "best guess" at the underlying relationship in the data, as we try to model that relationship using math.
+
+These models are just functions being graphed on top of the scatter plot, with the goal of minimizing the distance between the line and all the points on the plot. For straight-line relationships, these are _linear functions_ or "linear models". The straight-line graph of these models is sometimes called the "regression line" or the "predictor function", but you may have heard it called the @vocab{line of best fit}.
+
+When we make a model, we want it to be the closest possible approximation of all the points. A "good fit" has most of the points very close to the line, and a "bad fit" has the points very far away.
+
+You may already be familiar with the different forms of linear models:
+
+[cols="^1,^1,^1", options="header"]
+|===
+| Standard Form 	| Point-Slope Form			| Slope-Intercept Form
+| @math{Ax+By = C}	| @math{y-y_1 = m(x-x_1)}	| @math{y = mx+b}
+|===
+
+Depending on what part of the model we care about most, we might choose to use one display over another.
+
+- Standard Form makes it easy to find the x- and y-intercepts of the line
+- Point-Slope Form makes it easy to find the equation of the line given a single point and slope
+- Slope-Intercept Form makes it really easy to read off the slope and y-intercept
+
+Since we're looking for both the slope and the y-intercept for this model, it makes sense to use Slope-Intercept form.
+
+
+=== Investigate
+
+[.lesson-instruction]
+- Complete the first section ("Build a Model from Samples") in @printable-exercise{model-college-v-income.adoc}.
```

Comment:
FYI - I would expect that students will neither remember how to calculate slope, nor remember/know how to calculate y-intercept from 2points.

Reply:
In an Algebra 2 class?

Reply:
Fixed

Reply:
this is a huge improvement!